Index: axios/axios.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import axios from 'axios'\r\n\r\nconst baseUrl = \"https://localhost:3001/register\";\r\nconst getData = () => {\r\n    axios.get(baseUrl)\r\n        .then(response => {\r\n            console.log(response)\r\n        });\r\n}\r\n\r\nconst sendData = (req, response) => {\r\n    const body = req.body\r\n\r\n    if(!body.AccountName){\r\n        return response.status(400).json({\r\n        })\r\n    }\r\n\r\n    axios.post(baseUrl, {\r\n        AccountName: body.AccountName,\r\n        AccountType: body.AccountType,\r\n        Balance: body.Balance,\r\n        BalanceDate: body.BalanceDate,\r\n        UserID: body.UserID\r\n    })\r\n        .then(response =>{\r\n            console.log(response)\r\n        })\r\n}\r\n\r\nmodule.exports = {getData,sendData}
===================================================================
diff --git a/axios/axios.js b/axios/axios.js
--- a/axios/axios.js	
+++ b/axios/axios.js	
@@ -9,7 +9,7 @@
 }
 
 const sendData = (req, response) => {
-    const body = req.body
+    const body = req.params.id
 
     if(!body.AccountName){
         return response.status(400).json({
