Index: budgeting_app/src/axios/user.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import axios from \"axios\";\r\n\r\nconst baseUrl = \"https://localhost:3001/user/register\";\r\n\r\nconst sendData = async (req) => {\r\n    console.log(req)\r\n    const config = { 'content-type': 'application/json'}\r\n\r\n    await axios.post(baseUrl, req)\r\n\r\n        /*\r\n        .then(function (res) {\r\n            console.log(res)\r\n        })\r\n        .catch(function (res) {\r\n            console.log(res)\r\n        })\r\n\r\n         */\r\n}\r\n\r\n\r\nexport default sendData;
===================================================================
diff --git a/budgeting_app/src/axios/user.js b/budgeting_app/src/axios/user.js
--- a/budgeting_app/src/axios/user.js	
+++ b/budgeting_app/src/axios/user.js	
@@ -1,23 +1,18 @@
 import axios from "axios";
 
-const baseUrl = "https://localhost:3001/user/register";
+const baseUrl = "http://localhost:3001/user/register";
 
 const sendData = async (req) => {
     console.log(req)
     const config = { 'content-type': 'application/json'}
 
     await axios.post(baseUrl, req)
-
-        /*
         .then(function (res) {
             console.log(res)
         })
         .catch(function (res) {
             console.log(res)
         })
-
-         */
 }
 
-
 export default sendData;
\ No newline at end of file
